Core/sia380/sia380.py

```diff
@@ -0,0 +1,348 @@
+        Vdot_e_spec  Aussenluft-Volumenstrom [m3/m2h]
+        Vdot_inf_spec Aussenluft-Volumenstrom durch Infiltration [m3/m2h]
+        eta_rec Nutzungsgrad der Wärmerückgewinnung [-]
+        phi_P   Wärmeabgabe Personen [W/m2]
+        phi_L   Wärmeabgabe Beleuchtung [W/m2]
+        phi_A   Wärmeabgabe Geräte [W/m2]
+        t_P     Vollaststunden Personen [h]
+        t_L     Vollaststunden Beleuchtung [h]
+        t_A     Vollaststunden Geräte [h]
+        g       g-Wert [-]
+        f_sh    Reduktionsfaktor solare Wärmeeinträge [-]
+        I       description": "Solare Strahlung [Wh/m2]
+    """
+
+    """
+    cases according to SIA 2024:2015.
+    The norm defines following ROOM and/or BUILDING types (p.7-8):
+        _____________________________________________________________________________________
+              | abbr.     | description                                    |    code SIA 380
+        ______|___________|________________________________________________|_________________
+        - 1.1   mfh:        multi-family house (Mehrfamilienhaus)                   HNF1
+        - 1.2   efh:        single family house (Einfamilienhaus)                   HNF1
+        - 2.1   ...:        Hotelroom                                               HNF1
+        - 2.2   ...:        Hotel lobby                                             HNF1
+        - ...   ...:
+        - 3.1   office:     small office space (Einzel,- Gruppenbüro)               HNF2
+        - ...   ...:
+        - 4.1   school:     school room (Schulzimmer)                               HNF5
+        - ...   ...:
+        _____________________________________________________________________________________
+    """
+
+    f_sh = 0.9  # sia2024, p.12, 1.3.1.9 Reduktion solare Wärmeeinträge
+
+    tau = room["Zeitkonstante"]
+    theta_i_summer = room["Raumlufttemperatur Auslegung Kuehlung (Sommer)"]
+    theta_i_winter = room["Raumlufttemperatur Auslegung Heizen (Winter)"]
+
+    U_op = room["U-Wert opake Bauteile"]
+    U_w = room["U-Wert Fenster"]
+    Vdot_e_spec = room["Aussenluft-Volumenstrom (pro NGF)"]
+    Vdot_inf_spec = room["Aussenluft-Volumenstrom durch Infiltration"]
+    eta_rec = room["Temperatur-Aenderungsgrad der Waermerueckgewinnung"]
+    phi_P = room["Waermeeintragsleistung Personen (bei 24.0 deg C, bzw. 70 W)"]
+    phi_L = room["Waermeeintragsleistung der Raumbeleuchtung"]
+    phi_A = room["Waermeeintragsleistung der Geraete"]
+    t_P = [room["Vollaststunden pro Jahr (Personen)"]] * 12
+    t_L = [room["Jaehrliche Vollaststunden der Raumbeleuchtung"]] * 12
+    t_A = [room["Jaehrliche Vollaststunden der Geraete"]] * 12
+    g = room["Gesamtenergiedurchlassgrad Verglasung"]
+
+    # transforming daily sia2024 data to monthly
+    for i in range(len(dayspermonth)):
+        t_P[i] *= dayspermonth[i] / 365.0
+        t_L[i] *= dayspermonth[i] / 365.0
+        t_A[i] *= dayspermonth[i] / 365.0
+
+    return tau, theta_i_summer, theta_i_winter, \
```

Comment:
consider making this a type (either a class or a dict) so you can avoid multiline breaks. btw: inside parenthesis, you don't need the line breaks - so just return a tuple. it's a tuple anyway. except really do consider a class / named tuple or dictionary.

Reply:
ok yes I agree. I'll keep it nevertheless for this PR, because issue #281 will introduce some changes here. It will make sia380.py read building elements (containing individual construction properties), instead of the room json.